npm install fails with react 17.0.1 and react-native 0.64.0

**npx react-native init MyTestApp**
**cd MyTestApp && npm install @react-native-picker/picker --save**

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: xx@0.0.1
npm ERR! Found: react@17.0.1
npm ERR! node_modules/react
npm ERR!   react@"17.0.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.0" from @react-native-picker/picker@1.9.12
npm ERR! node_modules/@react-native-picker/picker
npm ERR!   @react-native-picker/picker@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/xx/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/xx/.npm/_logs/2021-03-15T11_31_38_410Z-debug.log

| software              | version
| --------------------- | -------
| react-native          | 0.64.0
| react-native-paper    | 4.7.2
| node                  | v14.15.1
| npm            | 7.6.3

--- a/package.json
+++ b/package.json
@@ -41,7 +41,7 @@
   "author": "",
   "license": "MIT",
   "peerDependencies": {
-    "react": "^16.0",
+    "react": "16 || 17",
     "react-native": ">=0.57"
   },
   "private": false,

